contrib/pyln-client/pyln/client/gossmap.py

```diff
@@ -52,9 +52,25 @@ def __init__(self, channel: 'GossmapChannel', direction: int,
         self.fee_base_msat: int = fields['fee_base_msat']
         self.fee_proportional_millionths: int = fields['fee_proportional_millionths']
 
+        # Cache the _scidd and hash to have faster operation later
+        # Unfortunately the @final decorator only comes for python3.8
+        self._scidd = f"{self.channel.scid}x{self.direction}"
```

Comment:
We've always used `/` IIRC to separate `scid` from `direction`, since `scid` already uses `x` as separator.